l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -109,3 +113,35 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
+
+    def __repr__(self) -> str:
+        fields = [
+            "Num Scenes",
+            "Num Frames",
+            "Num Agents",
+            "Total Time (hr)",
+            "Avg Frames per Scene",
+            "Avg Agents per Frame",
+            "Avg Scene Time (sec)",
+            "Avg Frame frequency",
+        ]
+        if len(self.agents) > 1:
+            times = self.frames[1:10]["timestamp"] - self.frames[0:9]["timestamp"]
+            frequency = np.mean(1 / (times / 1e9))  # from nano to sec
+        else:
+            frequency = 0  # can't read the frequency
+
+        values = [
+            len(self.scenes),
+            len(self.frames),
+            len(self.agents),
+            len(self.frames) / max(frequency, 1e-9) / 3600,
```

Comment:
Minor: max(..., 1e-9) is for catching div by 0? personally do not like it too much that the resulting value after the division then is a high value ... would prefer to get something like 0 - or what is the content of the zarr file then?

Reply:
I'll see what I can do